### DISCUSSION TITLE
[suggestion] Possibility to increase playback speed

### DISCUSSION THREAD

**Discussion by alekskomp** [Syncplay/syncplay#443] [Ideas]
Hi
Can you please add possibility to increase playback speed? 
x1.25 x1.5 x1.75 etc.
Or add syncing speed changes

**Comment by daniel-123** [comment]
Adding "proper" support for playback speed synchronization is considerably large undertaking in current code base.

That said - if you disable the "Slow down on minor desync" option, then Syncplay will not mess with playback speed of your player. If everybody in the room sets the same playback speed manually, then syncing should work just as if at playback rate of 1.

**Comment by Et0h** [comment]
I can see why people want this feature, but I agree with @daniel-123 that it would take quite a lot of work to implement and test as a lot of Syncplay's code is based on an assumption of playback being 1 second per second. This is one of those things that could end up inadvertently creating lots of bugs and so would need a lot of testing.

**Comment by NilsIrl** [comment]
#115 has been added to: https://syncplay.pl/about/ideas/

**Comment by Et0h** [comment]
As noted at #406:
> It's possible, but it isn't easy and would need a lot of testing to ensure it works across all media players, is compatible with managed rooms, changing rooms mid-playback, joining mid-playback (when first loading the file or following a disconnect when the file is already open), and how to handle slowdowns. You would also need a system to determine if some people in the room have an older version of Syncplay to warn them that they will be out of sync. Whoever takes on this project would need to be willing to maintain the feature in the future, which hopefully won't be as difficult as implementing it in the first place.
> 
> https://syncplay.pl/about/ideas/ lists "Allowing rooms to have a custom playback rate" as something that "Would need someone to come on board to develop". To date, nobody has volunteered so it isn't happening.
